pyboy/plugins/base_plugin.py

```diff
@@ -264,6 +264,9 @@ def game_area(self):
             if 0 <= _y < height and 0 <= _x < width:
                 tiles_matrix[_y][_x] = self.mapping[
                     s.tile_identifier] + self.sprite_offset # Adding offset to try to seperate sprites from tiles
+                if _y + 1 < height and len(s.tiles) == 2:
+                    tiles_matrix[_y + 1][_x] = self.mapping[s.tile_identifier + 1] # Adding second tile
```

Comment:
Nice catch! I think we need to handle the corner case where a sprite is placed above the screen, and only the lower tile is visible. That would have been filtered out by the `0 <= _y` above.

In `lcd.py` we check that the scanline is between `y` and `y+16`.

Reply:
In the latest pull, it's solved at least for the case where the indicated game area cuts the sprite in half. However, I'm not sure if a sprite can be placed at position -1 if the bounds are defined by the screen itself.

Reply:
Perfect! I've added a test case which demonstrates that it works too (you're still author of the commits and get credit).

I had to pull in some changes from my WIP branch, as the formatter I used is no longer supported.